### DISCUSSION TITLE
关于load的问题

### DISCUSSION THREAD

**Discussion by 131452099900** [milvus-io/milvus#30521] [Q&A and General discussion]
我有一些关于loadCollection的疑问，就是loadCollection是应该会去对象存储的index file去加载索引把，这个加载过程是不是比较耗损时间和资源，如果每次查询都这样操作是会导致性能的下降么。
1oad是否会存在并发的问题，就是如果没在应用层添加控制是有可能在一个线程准备操作的时候，然后另一个线程对collection进行release的会导致准备操作的线程失败吧。

**Comment by xiaofan-luan** [comment]
查询不会加载，加载时一次性的操作

**Reply by 131452099900** [reply]
额，我是业务上需要每次查询然后加载的然后release的，因为我这边的collection也比较多，不知道如果load进来不release的话会不会不太好，或者只在创建load一次？

**Reply by xiaofan-luan** [reply]
正常应该是load一次就好了
如果不是经常用的可以考虑mmap加载

**Reply by 131452099900** [reply]
可以load非mmap和mmap加载一起根据场景使用么，也就是同时存在根据需求选择

**Reply by xiaofan-luan** [reply]
2.3.8版本已经支持了表级别的mmap参数
